010-Getting-started/010-what-is-xata.mdx

```diff
@@ -1,300 +1,127 @@
-- it grows with you, in both scale and required functionality because it is based on proven data technologies, each best-in-class for the problems they solve.
+Access to the underlying data is provided in two main ways: via the PostgreSQL wire protocol, which is mediated by our SQL Proxy, and via our SDKs, which provide a higher-level interface to the data.
 
-Currently, the Xata service uses PostgreSQL as the data store, Elasticsearch as the search/analytics engine, and Kafka for storing the logical replication events. It also offers support for edge-caching. You can read more about how Xata works in this document and a more detailed explanation of the serverless aspect in this [blog post](https://xata.io/blog/what-is-a-serverless-database).
+![PostgreSQL Data Platform](images/postgresql_data_platform.png)
 
-In the future, we plan to also add built-in support for centralized caching (for example, Redis), block storage, better support for sharding between DBs, automatic global distribution, and so on.
+Individuals and indie developers like Xata because it is easy to get started, has a generous free-tier, offers a nice development experience, and solves common problems in a developer-first way (e.g. file attachments, full-text search, vector search).
 
-This type of data architecture, where a relational DB is used as the primary store and the data in it is replicated in other more specialized stores (search engine, data warehousing, business intelligence systems, etc.) is very popular among companies over a certain size. However, it requires significant amounts of glue code, operational overhead, monitoring, and expertise. Not anymore, Xata packages it all in a single opinionated service.
+Companies and larger teams like Xata because it is cost-efficient, consolidates tooling and reduces the operation load. Branching and zero-downtime migration workflows means that your team can iterate faster when performing complex schema changes.
 
-## How is it Different from Backend-as-a-Service (BaaS)?
+## PostgreSQL, the Xata way
 
-Backend-as-a-Service type of service is similar because it also offers the functionality of a serverless DB with more services on top of it. Firebase popularized the model already more than 10 years ago, and more recently players like Supabase, AppWrite, NHost, and others are offering open-source BaaS.
+As per the [StackOverflow survey of 2023](https://survey.stackoverflow.co/2023/#section-most-popular-technologies-databases), PostgreSQL is currently the most popular database system on the planet. Started in 1983, the PostgreSQL open source project has a long history of reliability and a huge and growing community. For many developers, PostgreSQL is the first choice for a database system, because of its ACID transactions, planner performance, JSON support, extensibility, and much more.
```

Comment:
```suggestion
As per the [StackOverflow survey of 2023](https://survey.stackoverflow.co/2023/#section-most-popular-technologies-databases), PostgreSQL is currently the most popular database system on the planet. Started in 1983, the PostgreSQL open source project has a long history of reliability and an ever growing community. For many developers, PostgreSQL is the first choice for a database system, because of its ACID transactions, planner performance, JSON support, extensibility, and much more.
```